docs/guide/ingress/annotations.md

```diff
@@ -16,6 +16,7 @@ You can add annotations to kubernetes Ingress and Service objects to customize t
 ## Annotations
 |Name                       | Type |Default|Location|MergeBehavior|
 |---------------------------|------|-------|--------|------|
+|[alb.ingress.kubernetes.io/name](#name)|string|N/A|Ingress|Exclusive|
```

Comment:
i think `name` is too vague here. how about `alb.ingress.kubernetes.io/load-balancer-name`? which also aligns with `alb.ingress.kubernetes.io/load-balancer-attributes`

Reply:
sounds good.